### DISCUSSION TITLE
"Missing Mass List in Scan __" Error after trying ADAP Resolver

### DISCUSSION THREAD

**Discussion by alishagreene** [mzmine/mzmine#1576] [Q&A]
I'm using MZMine 3.9.0 on Windows 11. 

I have Thermo HR-LC-MS (orbitrap) data. I used MSConvert to centroid the data, ensuring Peak Picking was selected. 

No matter what parameters I set (noise level, required intensity, etc.), I get the following error when trying to resolve chromatograms using ADAP resolver:
![MZMine_Error](https://github.com/mzmine/mzmine3/assets/155596540/b5c64f02-9369-40e0-9554-b65bac6ba220)

It also looks like this at the bottom:
![MZMine_Error_2](https://github.com/mzmine/mzmine3/assets/155596540/0c97496c-1dd7-49fe-a957-2a320cc712fc)

Has anyone had this happen? The weird thing is that I've done a workflow with this exact data before and did not have this issue. I've even tried analyzing in profile mode, too, with the original data just converted from raw to mzml using MSConvert. 

Thank you in advance!

**Comment by kyrasolaris01** [comment]
Hi,

Looks like you didn't run mass detection on MS 2 level.

**Comment by arpita-007** [comment]
Hi @alishagreene were you able to resolve this error? I am also facing the same issue. @kyrasolaris01 I tried your suggestion also, but it's still showing the same error.

**Reply by kyrasolaris01** [reply]
Hi,
very difficult to determine if everything is setup correctly without getting information of what you have tried exactly. Before going into detail I suggest you try processing your data with the wizard. The wizard ensures all necessary steps are run in the correct order: https://mzmine.github.io/mzmine_documentation/wizard.html
I hope that helps.

**Comment by arpita-007** [comment]
Hi,
Thank you so much for the prompt response. I tried disabling the ms/ms
pairing and the error disappeared. But I am not sure whether that's the
correct way to do it or not.
Surely, I will try the wizard.
Thanks again for your help!


On Wed, Feb 21, 2024 at 4:38 PM kyrasolaris01 ***@***.***>
wrote:

> Hi,
> very difficult to determine if everything is setup correctly without
> getting information of what you have tried exactly. Before going into
> detail I suggest you try processing your data with the wizard. The wizard
> ensures all necessary steps are run in the correct order:
> https://mzmine.github.io/mzmine_documentation/wizard.html
> I hope that helps.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mzmine/mzmine3/discussions/1576#discussioncomment-8542162>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZFL7BOBWONG7ME2C22H67TYUXIR7AVCNFSM6AAAAABBMHLTN2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKNBSGE3DE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Regards!
Arpita Singh
Ph.D. Student
c/o Dr. Pallavi Kshetrapal
Translational Health Science and Technology Institute
NCR Biotech Science Cluster, 3rd Milestone, Faridabad – Gurgaon Expressway,
Faridabad, Haryana 121001
Phone no.: 0129-2876363
